### DISCUSSION TITLE
Feature:  Add support for MIDI controllers

### DISCUSSION THREAD

**Discussion by GordonSmith** [File-New-Project/EarTrumpet#666] [Ideas]
Allow changing the volume of individual apps via midi controller!

**Comment by schmic** [comment]
Oh yes please!

**Comment by jacky4566** [comment]
+1

It would also be fantastic to pull the program icons through IAudioSessionControl::GetIconPath method.

I'm working on a midi controller with active display!

**Comment by flibble666** [comment]
Sorry to revive an old post but i found [this ](https://github.com/ho0ber/NK2Tray) it allows you to control Windows Audio Mixer with a Korg nanoKONTROL2 and other similar control surfaces.
and i would assume work with ear trumpet.

**Reply by Yuzam** [reply]
NK2Tray only allows you to use 2 midi controllers models, nano and the Berhinger nano, there are hundreds of MIDI controllers in the market. I am using Voicemeeter and not EarTrumpet because it allows me to use my old BCF2000 and my X-Touch Com­pact. I would start using EarTrumpet for sure if i could use my controllers.
